cli/command/run/cmd.go

```diff
@@ -103,11 +111,57 @@ func newRunner(p config.Project, cli *command.SauceCtlCli, seq fleet.Sequencer)
 		if err != nil {
 			return nil, err
 		}
+
+		cip := createCIProvider()
+		seq := createCISequencer(p, cip)
+
 		log.Info().Msg("Starting CI runner")
-		return ci.NewRunner(p, cli, seq, rc)
+		return ci.NewRunner(p, cli, seq, rc, cip)
 	}
 	log.Info().Msg("Starting local runner")
-	return docker.NewRunner(p, cli, seq)
+	return docker.NewRunner(p, cli, &memseq.Sequencer{})
+}
+
+func createCIProvider() ci.Provider {
+	enableCIProviders()
+	cip := ci.Detect()
+	// Allow users to override the CI build ID
+	if ciBuildID != "" {
+		log.Info().Str("id", ciBuildID).Msg("Using user provided build ID.")
+		cip.SetBuildID(ciBuildID)
+	}
+
+	return cip
+}
+
+func createCISequencer(p config.Project, cip ci.Provider) fleet.Sequencer {
+	if !p.Parallel {
+		log.Info().Msg("Parallel execution is turned off. Running tests sequentially.")
```

Comment:
@alexplischke do we want to add a link to Documentation here in the future? Maybe add a TODO?

Reply:
Kinda like a pointer for the user to actually start using this feature? Might be a good idea, a link to the README for instance.